### ISSUE TITLE
disable Submit button form submit

### ISSUE THREAD

**Issue by EuphoSomit** [formvalidation/support#119]
I am always enabling the submit button. The problem is that, if the field have more than one validation check and if first validation is not valid. Then, when you click on submit button, it's check the last validation and will give the error of last check. As previously, it should stop with the first validation check itself.

**Comment by nghuuphuoc** [comment]
> Then, when you click on submit button, it's check the last validation and will give the error of last check. As previously, it should stop with the first validation check itself

By default, the plugin will execute all validators used by field. If you want to stop right after the first one which the field doesn't pass, then use the [verbose](http://formvalidation.io/settings/#form-verbose) option.

**Comment by EuphoSomit** [comment]
Yes, I am using the verbose option. At the time of filling the form, it's working but when I submit the form, it's not.
